pages/profile/switch-user.tsx

```diff
@@ -0,0 +1,56 @@
+import Head from 'next/head';
+import { useTranslation } from 'next-i18next';
+import UserLayout from '../../src/features/common/Layout/UserLayout/UserLayout';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import SwitchUser from '../../src/features/user/Settings/SwitchUser';
+import { UserPropsContext } from '../../src/features/common/Layout/UserPropsContext';
+import { useContext } from 'react';
+import AccessDeniedLoader from '../../src/features/common/ContentLoaders/Projects/AccessDeniedLoader';
+
+const SwitchUserPage = () => {
+  const { user } = useContext(UserPropsContext);
+  const { t } = useTranslation('me');
+
+  return (
+    <UserLayout>
+      <Head>
+        <title>{t('me:switchUser')}</title>
+      </Head>
+      {user?.allowedToSwitch ? <SwitchUser /> : <AccessDeniedLoader />}
```

Comment:
@sunilsabatp update the User type in UserPropsContext to include the `allowedToSwitch` property.